### PR TITLE
doc: Add Milvus standalone all in one script

### PR DESCRIPTION
**Pull request by pingliu**

### CODE REVIEW

File: scripts/standalone_embed.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        -v ./embedEtcd.yaml:/milvus/configs/embedEtcd.yaml \
+        -p 19530:19530 \
+        -p 9091:9091 \
+        -p 2379:2379 \
+        --health-cmd="curl -f http://localhost:9091/healthz" \
+        --health-interval=30s \
+        --health-start-period=90s \
+        --health-timeout=20s \
+        --health-retries=3 \
+        milvusdb/milvus:latest \
```
Comment (LoveEachDay) [inline scripts/standalone_embed.sh:42]: We should use specific image tag instead of `latest`.

Comment (pingliu) [reply scripts/standalone_embed.sh]: already changed

File: scripts/standalone_embed.sh [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    then
+        echo "Stop failed."
+        exit 1
+    fi
+    echo "Stop successfully."
+
+}
+
+delete() {
+    sudo docker rm milvus-standalone 1> /dev/null
```
Comment (LoveEachDay) [inline scripts/standalone_embed.sh:77]: We'd better add a remainder for user when `delete` without `stop`.

**Review by LoveEachDay — APPROVED** [review]
/lgtm

**Pull request merged**